lib/item_repository.rb

```diff
@@ -0,0 +1,25 @@
+require_relative './item'
+require_relative './database_connection'
+
+class ItemRepository
+  def all
+    sql = 'SELECT id, name, price, quantity FROM items;'
+    result_set = DatabaseConnection.exec_params(sql, [])
```

Comment:
Could have 'sql_param = [ ]' for consistency/readability with the 'create' method